verify.py

```diff
@@ -0,0 +1,25 @@
+import sys
+import os
+from importlib import import_module
+
+EXERCISES = {
+    "00": import_module("setup.verify_env"),
+    "01": import_module("setup.01-manual_testing"),
+    "02": import_module("setup.02-robot_syntax"),
+    "03": import_module("setup.03-keywords_and_variables"),
+    "04": import_module("setup.04-setups_and_teardowns"),
+    "05": import_module("setup.05-negative_testing"),
+    "06": import_module("setup.06-resource_files"),
+    "07": import_module("setup.07-test_template")
```

Comment:
eg:
```python
EXERCISES = {
    0: import_module("setup.verify_env"),
    1: import_module("setup.01-manual_testing"),
    # ...
}

def main(exercise):
     module = EXERCISES[int(exercise)]

Reply:
Our exercises still use 00-07 in *.md files https://github.com/eficode-academy/rf-katas/tree/update_win_exercises/exercises so that was the reason to use that.

Reply:
Yes, but did you read my comment?

Reply:
@EfiGaja  &/ @asimell which one do you prefer better? For me it's ok to change from dict to list and then use e.g `python verify.py 0`

Reply:
Ah no it seems that I didn't read it fully. I'll try out the outcome

Reply:
Changed the behaviour in https://github.com/eficode-academy/rf-katas/pull/29/commits/1dbf4a806fac6e33e50ed41e6561ff8cc8b0c279